docs/validatorctl/1-download-and-install.md

```diff
@@ -0,0 +1,56 @@
+---
+title: Download & Install
+sidebar_position: 1
+---
+
+## MacOS & Linux
+
+### Installing from release binaries
+
+Download and install the latest release for your OS and CPU architecture from [GitHub Releases](https://github.com/validator-labs/validatorctl/releases).
+
+You can use [this helper script](https://raw.githubusercontent.com/validator-labs/docs/main/scripts/get_validator.sh) to install the latest version of the binary to `/usr/local/bin/validatorctl`:
+
+```shell
+curl -L https://raw.githubusercontent.com/validator-labs/docs/main/scripts/get_validator.sh | bash
+```
+
+### Installing with `go install`
+
+To install, run:
+
+```shell
+go install github.com/validator-labs/validatorctl@latest
+```
+
+:::note
+
+The MacOS build is currently only available for arm64 and the Linux build is currently only available for amd64.
+
+:::
```

Comment:
This shouldnt be only mac/linux specific. Im pretty sure if you're on windows you can still run go install, and then based on your `GOOS` and `GOARCH` it'll setup the binary correctly (ie as a .exe and) and put it in your PATH.